configs/HGLRC_H743_PRO/config.h

```diff
@@ -0,0 +1,144 @@
+#define SPI2_SDO_PIN            PB15
+#define MAX7456_SPI_CS_PIN      PB12
+#define MAX7456_SPI_INSTANCE    SPI2
+
+#define SPI3_SCK_PIN            PB3
+#define SPI3_SDI_PIN            PB4
+#define SPI3_SDO_PIN            PB2
+#define FLASH_CS_PIN            PA15
+#define FLASH_SPI_INSTANCE      SPI3
+#define DEFAULT_BLACKBOX_DEVICE BLACKBOX_DEVICE_FLASH
+
+#define BEEPER_PIN              PE5
+#define MOTOR1_PIN              PE9
+#define MOTOR2_PIN              PE11
+#define MOTOR3_PIN              PE13
+#define MOTOR4_PIN              PE14
+#define MOTOR5_PIN              PD12
+#define MOTOR6_PIN              PD13
+#define MOTOR7_PIN              PD14
+#define MOTOR8_PIN              PD15
+#define LED_STRIP_PIN           PA0
+#define SERVO1_PIN              PA2
+#define SERVO2_PIN              PA3
+#define SERVO3_PIN              PB0
+#define SERVO4_PIN              PB1
+
+#define UART1_TX_PIN            PA9
+#define UART1_RX_PIN            PA10
+#define UART2_TX_PIN            PD5
+#define UART2_RX_PIN            PD6
+#define UART3_TX_PIN            PD8
+#define UART3_RX_PIN            PD9
+#define UART4_TX_PIN            PD1
+#define UART4_RX_PIN            PA1
+#define UART5_TX_PIN            PB6
+#define UART5_RX_PIN            PB5
+#define UART7_TX_PIN            PE8
+#define UART7_RX_PIN            PE7
+#define UART8_TX_PIN            PE1
+#define UART8_RX_PIN            PE0
+
+#define I2C1_SCL_PIN            PB8
+#define I2C1_SDA_PIN            PB7
+#define I2C2_SCL_PIN            PB10
+#define I2C2_SDA_PIN            PB11
+
+#define LED0_PIN                PE4
+#define LED1_PIN                PE6
+#define LED2_PIN                PE3
+
+#define ADC_VBAT_PIN            PC0
+#define ADC_CURR_PIN            PC1
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP(0,  BEEPER_PIN,       1, -1) \
+    TIMER_PIN_MAP(1,  MOTOR1_PIN,       1,  0) \
+    TIMER_PIN_MAP(2,  MOTOR2_PIN,       1,  1) \
+    TIMER_PIN_MAP(3,  MOTOR3_PIN,       1,  2) \
+    TIMER_PIN_MAP(4,  MOTOR4_PIN,       1,  3) \
+    TIMER_PIN_MAP(5,  MOTOR5_PIN,       2,  4) \
+    TIMER_PIN_MAP(6,  MOTOR6_PIN,       2,  5) \
+    TIMER_PIN_MAP(7,  MOTOR7_PIN,       2, -1) \
+    TIMER_PIN_MAP(8,  MOTOR8_PIN,       2, -1) \
+    TIMER_PIN_MAP(9,  LED_STRIP_PIN,    1,  6) \
+    TIMER_PIN_MAP(10, SERVO1_PIN,       1, -1) \
+    TIMER_PIN_MAP(11, SERVO2_PIN,       1, -1) \
+    TIMER_PIN_MAP(12, SERVO3_PIN,       1, -1) \
+    TIMER_PIN_MAP(13, SERVO4_PIN,       1, -1) \
+    TIMER_PIN_MAP(14, GYRO_1_CLKIN_PIN, 1, -1)
+
+#define TIMUP1_DMA_OPT      9
+#define TIMUP5_DMA_OPT      10
+#define ADC1_DMA_OPT        11
+#define ADC3_DMA_OPT        12
+
+#define BARO_I2C_INSTANCE   I2CDEV_1
+#define MAG_I2C_INSTANCE    I2CDEV_2
+
+#define DEFAULT_CURRENT_METER_SOURCE    CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE    VOLTAGE_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SCALE     210
+
+#define PINIO1_PIN     PE2
+#define PINIO1_BOX     40
+#define PINIO1_CONFIG  129
```

Comment:
Should be low by default to select cam 1
```suggestion
#define PINIO1_CONFIG  1
```